tests/api2/update.py

```diff
@@ -90,6 +101,8 @@ def test_07_install_update():
 def test_08_verify_the_update_is_successful():
     if upgrade is False:
         pytest.skip('No update found')
+    elif download_hang is True:
+        pytest.skip('Download fail')
```

Comment:
This should be more descriptive, *which* download. In this case getting the trains failed.

Reply:
True